src/windows/WATNav.js

```diff
@@ -46,5 +69,37 @@ navigateBack = function (e) {
   return true;
 }
 
+createBackButton = function () {
+    var wrapperDiv = document.createElement("div");
+
+    wrapperDiv.id = "backbutton-wrapper";
+    wrapperDiv.style.zIndex = WAT.components.webView.style.zIndex + 100;
```

Comment:
Note that I used 100 for the app Bar, so its in the same z-index, we may want to have this behind the nav and app bars. I would change it to a number between WAT.components.webView.style.zIndex +1 and WAT.components.webView.style.zIndex + 99